### PR TITLE
Tests for vendors

### PR DESCRIPTION
**Pull request by RafailFerre**
Create specs: vendorCreate, vendorDelete, vendorGet, vendorGetAll, vendorUpdate.
Create vendorHelper.

Fixed various.spec - endPoint changed. 

> Last:  endPoint = str.body.payload.items[0].message.split('\n')[4].split('https://clientbase.us')[1];  **test fail**

> Now:  endPoint = str.body.payload.items[0].message.split('\n')[4].split('https://clientbase.pasv.us')[1];  **test pass**

Add scripts in package.json "lint" and "format" prettier. Add description
Some files was format prettier.

Changed: .env.example BASE_URL for better understanding.

Add negative scenarios for 'delete vendor'

### CODE REVIEW

**Review by dumpdusty — COMMENTED** [review]
LGTM, would you like to add a negative scenario for `delete vendor` feature?

**Comment by RafailFerre** [comment]
> LGTM, would you like to add a negative scenario for `delete vendor` feature?

Yes sir! I'll have make it today.

**Comment by RafailFerre** [comment]
> LGTM, would you like to add a negative scenario for `delete vendor` feature?

Done. I've added negative tests for 'delete vendor'

**Review by dumpdusty — DISMISSED** [review]

File: test/vendors/vendorDelete.spec.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ import { expect } from 'chai';
...
     describe('POSITIVE: DELETE VENDOR BY ID', () => {
         before(async() => {
 
         const vendorId = (await vendorHelper.createVendor(vendorHelper.vendorData)).body.payload
 
         res = await vendorHelper.deleteVendor(vendorId)
+        resGetVendor = await vendorHelper.getVendor(vendorId)
         });
 
         it('verify status code', async () => {
```
Comment (dumpdusty) [inline test/vendors/vendorDelete.spec.js:17]: looks like missing assertion

**Review by dumpdusty — APPROVED** [review]
Looks good so far

**Review by ZdanovichOleg — APPROVED** [review]

**Pull request merged**